admin_manual/enterprise_clients/update_branded_desktop_clients.rst

```diff
@@ -0,0 +1,199 @@
+       
+       https://mycloud.example.com/install/mycloud-2.1.1.787.pkg.tbz.sig
+   
+   -  You need a human-readable Website with links to your client
+      files. This could be https://mycloud.example.com/install/ with
+      ``Options +Indexes`` in your ownCloud ``.htaccess`` file.
+
+Install client-updater-server
+-----------------------------
+
+#. Download ``client-updater-server-0.3.tar.xz`` from
+   https://customer.owncloud.com/
+#. Extract ``client-updater-server-0.3.tar.xz`` to your Web server. The
+   ``index.php`` must be accessible at
+   ``https://mycloud.example.com/updates/index.php``
+#. Copy your ownCloud ``config/default.php`` file, and name it according 
+   your **Application short name** as configured in ownBrander.
+
+   -  Example:
+       ``config/mycloud.php``
+
+Configure client-updater-server
+-------------------------------
+
+All configuration is done in your ``config/mycloud.php``::
+
+    <?php
+
+    $updateInfo = [
+        'win32' => [
+            'currentVersion' => '2.1.1.240',
+            'currentVersionString' => 'MyCloud Client 2.1.1 (build 240)',
+            'updateUrl' => 'https://mycloud.example.com/install/',
+            'downloadUrl' => 
+             'https://mycloud.example.com/install/mycloud-2.1.2.240-setup.exe',
+        ],
+
+        'linux' => array(
+            'currentVersion' => '2.1.1',
+            'currentVersionString' => 'MyCloud Client 2.1.1',
+            'updateUrl' => 'https://mycloud.example.com/install/',
+        ),
+
+        'macos' => array(
+            'currentVersion' => '2.1.1.787',
+            'currentVersionString' => 'MyCloud Client 2.1.1 (build 787)',
+            'downloadUrl' => 
+             'https://mycloud.example.com/install/mycloud-2.1.1.787.pkg.tbz',
+            'pubDate' => '2016-02-23',
+            'signature' => 
+              'MCwCFFedScUKeRXYMS6vKVLw821B+/+lAhRFNXHSvB9GNHOuI5cw==',
+            'minimumSystemVersion' => '10.7.0',
+        ),
+
+    ];
+
+Windows
+^^^^^^^
+
+-  ``'currentVersion'``
+   Exact version of the new client, including the build nr
+-  ``'currentVersionString'``
+   Name of the new client, same as “Application name” configured in
+   ownBrander
+-  ``'updateUrl'``
+   Human-readable website with links to your new client files
+-  ``'downloadUrl'``
+   Full URL to download the \*.exe file. http\ **s** needed
+
+Linux
+^^^^^
+
+If you host a branded download repository for your users, then you don’t
+need to notify your users about new releases. Just configure an older
+version than the version you deployed to your users to never trigger a 
+notification.
+
+-  ``'currentVersion'``
+   Exact version of the new client, including the build nr
+-  ``'currentVersionString'``
+   Name of the new client, same as “Application name” configured in
+   ownBrander
+-  ``'updateUrl'``
+   Human-readable Website with links to your new client files to
+   manually install new client versions
+
```

Comment:
@michaelstingl That is a bit confusing-- what is the reason for disabling update notifications for Linux users? Does the Linux client have to be downloaded and installed manually, rather than using a package manager?